### DISCUSSION TITLE
Taailwind Daisyui cli

### DISCUSSION THREAD

**Discussion by coderdnewbie** [microsoft/fluentui-blazor#2782] [General]
The cli I mentioned in the meeting is here: 

https://github.com/dobicinaitis/tailwind-cli-extra

If there are any questions about getting tailwind to work with FluentUI theming I am happy to answer here.

**Comment by coderdnewbie** [comment]
no comments so closing

**Reply by vnbaaij** [reply]
I'd like to keep this open because it doesn't shop up in the overview list by default otherwise
